src/App.tsx

```diff
@@ -26,70 +26,128 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((a, b) => {
+      if (a > b) {
+        return 1;
+      }
+
+      if (a < b) {
+        return -1;
+      }
+
+      return 0;
+    });
```

Comment:
Use better naming insteab of a,b. For example good1, good2.
For sorting string you can use localeCompare()

```suggestion
    visibleGoods.sort((good1, good2) => {
      return good1.localeCompare(good2);
```